dkc/core/tests/test_folder.py

```diff
@@ -87,11 +85,9 @@ def test_folder_sibling_names_unique_files(file, folder_factory):
 
 @pytest.mark.django_db
 def test_root_folder_names_unique(folder, folder_factory):
-    other_root = folder_factory.build(name=folder.name)
```

Comment:
As the models' foreign key relationships have gotten more complex, the factory "build" strategy is becoming more and more unwieldy. I plan to introduce a follow-up PR to remove nearly all usage of the "build" strategy.